run

```diff
@@ -243,7 +243,7 @@ find "$judge/src" -name '*.java' \
 jar -cf "judge.jar" -C /tmp/build .
 
 # Compiling the tests
-if ! find "$resources" -name '*.java' | xargs javac -cp ".:${resources}:${worklibs}:${testlibs}:judge.jar" -d . -sourcepath "$resources" > "$compilation" 2>&1; then
+if ! find "$resources" -name '*.java' | xargs javac -Xdiags:verbose -cp ".:${resources}:${worklibs}:${testlibs}:judge.jar" -d . -sourcepath "$resources" > "$compilation" 2>&1; then
```

Comment:
Is this relevant to the PR? (It doesn't seem like it is.)

Reply:
It is, but I forgot to mention it in the commit message; I'll modify it and push after fixing some other stuff.